src/Azure.Functions.Cli/Helpers/AzureHelper.cs

```diff
@@ -326,8 +350,8 @@ public static async Task<Site> LoadSiteObjectAsync(Site site, string accessToken
             var url = new Uri($"{managementURL}{site.SiteId}?api-version={ArmUriTemplates.WebsitesApiVersion}");
             var armSite = await ArmHttpAsync<ArmWrapper<ArmWebsite>>(HttpMethod.Get, url, accessToken);
 
-            site.HostName = armSite.properties.enabledHostNames.FirstOrDefault(s => s.IndexOf(".scm.", StringComparison.OrdinalIgnoreCase) == -1);
-            site.ScmUri = armSite.properties.enabledHostNames.FirstOrDefault(s => s.IndexOf(".scm.", StringComparison.OrdinalIgnoreCase) != -1);
+            site.HostName = armSite.properties.enabledHostNames?.FirstOrDefault(s => s.IndexOf(".scm.", StringComparison.OrdinalIgnoreCase) == -1);
```

Comment:
Clarification point: are there scenarios now where we expect the `enabledHostName` to be `null`? If so, is it acceptable for the `site.HostName` and `site.ScmUri` to be `null`?

Reply:
The null check is probably not needed anymore. The `enabledHostNames` was coming as `null` earlier for function apps on Container Apps. They are continuously working the backend API. It is returning the list now.  It will takeout the null check as it is not needed anymore.